corehq/apps/reports/standard/project_health.py

```diff
@@ -327,6 +339,39 @@ def previous_six_months(self):
             last_month_summary = this_month_summary
         return six_month_summary
 
+    def export_summary(self, six_months):
+        return build_worksheet(title="Six Month Performance Summary",
+                               headers=['month', 'num_high_performing_users', 'num_low_performing_users',
+                                        'total_active', 'total_inactive', 'total_num_users'],
+                               rows=[[monthly_summary.month.isoformat(),
+                                      monthly_summary.number_of_performing_users,
+                                      monthly_summary.number_of_low_performing_users, monthly_summary.active,
+                                      monthly_summary.inactive, monthly_summary.total_users_by_month]
+                                     for monthly_summary in six_months])
```

Comment:
One last bit of (non actionable) feedback - this block here is a little confusing, because the first two `rows` have only one item on it, then the latter two have two items.  At a glance, it makes it look like there are only four elements in this thing.  This is quite indented, which forces your hand a bit - it might be better to define `rows` outside the function call.  You could also start the parameters on a newline:
```python
return build_worksheet(
    title="Six Month Performance Summary",
    ...
)
```